filters_test.go

```diff
@@ -311,6 +311,49 @@ func TestFirst(t *testing.T) {
 	}
 }
 
+func TestLast(t *testing.T) {
+	t.Parallel()
+	want, err := ioutil.ReadFile("testdata/last10.golden.txt")
+	if err != nil {
+		t.Fatal(err)
+	}
+	input := File("testdata/last.input.txt")
```

Comment:
Since this file is identical to `first.input.txt` I suggest we just re-use that file instead.